src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Maintaining Aspect Ratio in CSS'
```

Comment:
```suggestion
title: 'Maintaining aspect ratio in CSS'
```
Please do a search for `aspect` and check the following:

* When referring to the property, use `aspect-ratio` (formatted as inline code)
* When referring to the concept, use "aspect ratio" (omit the quotes, no dash, keep it lowercase)

Reply:
How do you feel about changing the title to: "The new way to maintain aspect ratio in CSS"